modules/shared/src/main/scala/io/constellationnetwork/currency/schema/currency.scala

```diff
@@ -162,6 +162,7 @@ object currency {
     dataApplication: Option[DataApplicationPart] = None,
     messages: Option[SortedSet[Signed[CurrencyMessage]]] = None,
     feeTransactions: Option[SortedSet[Signed[FeeTransaction]]] = None,
+    spendTransactions: Option[SortedSet[SpendTransaction]] = None,
```

Comment:
Does the SpendTransaction should be a part of Currency snapshot?

Reply:
Yes
https://constellationnetwork.atlassian.net/browse/PROT-843

Reply:
@marcinwadon we need to align how we will populate this. We need to somehow extract this in the output of `combine` function.

Reply:
I know the ticket but it may be outdated. I think the most recent one is the sequence diagram we've discussed together on the meeting. We're still discussing how metagraph would scan the network, so it may be that it would have to be aggregated on GL0 or maybe we would have more than one tote for marking the swap. I'd wait with it until we have the process well described

Reply:
I mean, I'd like to make schema update once, so let's make sure it's correct and covers our flow well

Reply:
Re: how to populate - maybe it needs to go a different path than the combine fn -> enqueued directly as an event to currency consensus, as allow spends on GL0.

Reply:
Got it @marcinwadon . @ryle-goehausen has described that we will have the SpendTransaction in GlobalSnapshots and CurrencySnapshots. So, I think we can wait, what do you think @ryle-goehausen ?